src/components/Audioplayer/ScrollButton/index.js

```diff
@@ -1,9 +1,19 @@
 import React, { PropTypes } from 'react';
+import styled from 'styled-components';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Tooltip from 'react-bootstrap/lib/Tooltip';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-const style = require('../style.scss');
+const ScrollLink = styled.a`
+  width: 100%;
+  display: inline-block;
+  cursor: pointer;
+  padding-right: 1.5%;
+  color: ${props => (props.active ? props.theme.brandPrimary : props.theme.textColor)};
+  outline: none;
+  cursor: pointer;
+  margin-bottom: 0;
+`;
```

Comment:
Or
```
const style = X`
color: yellow;
`
<div className={style} /> 
```

But the magic in styled components is having props affect css properties and the theming is powerful!